tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientHelper.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper;
+
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+/**
+ * Helper that provides operations to manage the CLI operations via {@link OnlineManagementClient} API.
+ */
+public class ManagementClientHelper {
+
+    /**
+     * Executes a valid CLI command through the given {@link OnlineManagementClient} instance.
+     *
+     * @param client  {@link OnlineManagementClient} instance used to execute the command
+     * @param command A valid CLI command
+     * @return {@link ModelNodeResult} instance to represent the command execution outcome
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
```

Comment:
What is the reason for wrapping the exception?

Reply:
The reason is that I believe it's good practice that will pay later, e.g. in case we should expect for a given type of exception.